### PR TITLE
RMC-112: Change acceptance test file filter for Census filenames.

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
RMC-112 Create initial England print file.

# What has changed
Change to file name filter.

# How to test?
Checkout branch rmc-112 for Action Service and ActionExporter Service. 
Build the services with Maven 'mvn clean install'
Make sure new docker images are produced
If not run docker in the respective repository e.g 'docker build -t eu.gcr.io/census-rm-ci/rm/census-rm-actionexportersvc .'

### CODE REVIEW

**Comment by rjweeks** [comment]
[RMC-112 Action Exporter Service to generate initial contact letter Print File England [8]](https://trello.com/c/1l3Y23Pj/593-rmc-112-action-exporter-service-to-generate-initial-contact-letter-print-file-england-8)

**Review by AdamHawtin — APPROVED** [review]
Passed for me locally 👍

**Review by hbrace — APPROVED** [review]
Passed for me!

**Review by AdamHawtin — DISMISSED** [review]
Needs updating with the `request_type` directory for the print files.

**Review by jcox-dev — CHANGES_REQUESTED** [review]
Wasn't sure if I was to use the (now closed) PR branch of the action service but I can't seem to get the tests passing with the changes that (only) exist in here and the exporter.

`files_found=[]`

**Review by jcox-dev — APPROVED** [review]

File: acceptance_tests/features/steps/print_file_steps.py [hunk 1]
```diff
@@ -19,14 +19,14 @@ def check_correct_files_on_sftp_server(context):
 def _check_notification_files_have_all_the_expected_data(context, expected_csv_lines):
     with SftpUtility() as sftp_utility:
         _validate_print_file_content(sftp_utility, context.survey_ref, context.period,
-                                     context.test_start_datetime, expected_csv_lines)
+                                     context.test_start_local_datetime, expected_csv_lines)
 
 
 @retry(retry_on_exception=lambda e: isinstance(e, FileNotFoundError), wait_fixed=5000, stop_max_attempt_number=24)
 def _validate_print_file_content(sftp_utility, survey_ref, period, start_of_test, expected_csv_lines):
```
Comment (AdamHawtin) [inline acceptance_tests/features/steps/print_file_steps.py]: This function doesn't use the `survey_ref` or `period` parameters anymore.

**Pull request merged**